app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from '../redux/greetings/greeting';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state);
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+  return (
+    <h2>{greeting.message}</h2>
+  );
+};
+
+export default Greeting;
```

Comment:
- [ ] It would be better for your application to display a greeting randomly when Greetings is clicked, currently, a user has to refresh the whole app and go back to `http://127.0.0.1:3000` before they see another random greeting, see video below, please fix it so users can see random greetings when they click on greetings 👍🏿  

https://user-images.githubusercontent.com/69638013/204736048-bd9a71fc-49c0-4a48-908a-3a458c73b749.mp4